src/TitleBindPieces/TitleBindPiecesContainer.test.js

```diff
@@ -0,0 +1,85 @@
+import { MemoryRouter } from 'react-router-dom';
+
+import {
+  render,
+  screen,
+} from '@folio/jest-config-stripes/testing-library/react';
+
+import { useTitle } from '../common/hooks';
+import {
+  usePOLine,
+  useUnboundPieces,
+} from './hooks';
+import TitleBindPieces from './TitleBindPieces';
+import TitleBindPiecesContainer from './TitleBindPiecesContainer';
+
+jest.mock('@folio/stripes/core', () => ({
```

Comment:
[Possibly you don't need it](https://github.com/folio-org/ui-receiving/pull/534/files#diff-9afaa1ba5750c66453763dace92ff7ffed804e0f3baa1201594a4f87ddf9b399R63)